incubator/jenkins/master-image/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM jenkins:2.7.3
+RUN /usr/local/bin/install-plugins.sh kubernetes:0.8 workflow-aggregator:2.3
```

Comment:
In my testing, when we put the config file in place the upgrade wizard no longer runs.